client/modules/IDE/components/ErrorModal.stories.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import ErrorModal from './ErrorModal';
+
+export default {
+  title: 'IDE/ErrorModal',
+  component: ErrorModal,
+  argTypes: {
+    type: {
+      options: [
+        'forceAuthentication',
+        'staleSession',
+        'staleProject',
+        'oauthError'
+      ],
+      control: { type: 'select' }
+    },
+    service: {
+      options: ['google', 'github'],
+      control: { type: 'select' }
+    },
+    closeModal: { action: 'closed' }
+  }
+};
+
+const Template = (args) => <ErrorModal {...args} />;
```

Comment:
Oh interesting. I'm still familiarising myself with the components in the application. I was wondering where the overlay was 😄  It would make a nicer story seeing the modal represented with the overlay.

Reply:
Honestly it feels like a bit of a naming mistake that this is called `ErrorModal` when it doesn't include the modal part.  Really it's more of an `ErrorMessage`!

It could potentially be modified so that it takes the overlay's `title` and `ariaLabel` as props and renders both the `Overlay` and the message.